airbyte-integrations/connectors/source-intercom/integration_tests/integration_test.py

```diff
@@ -93,7 +93,7 @@ def test_companies_scroll(stream_attributes):
     # read all records
     records = []
     for slice in stream2.stream_slices(sync_mode=SyncMode.full_refresh):
-        records += list(stream2.read_records(sync_mode=SyncMode, stream_slice=slice))
+        records += list(stream2.read_records(sync_mode=SyncMode.full_refresh, stream_slice=slice))
```

Comment:
Is this only a typo fix?

Reply:
Yea, figured I'd sneak it in. Happy to remove if you'd prefer not to pollute the PR.